central/complianceoperator/v2/checkresults/service/service_impl.go

```diff
@@ -49,16 +52,19 @@ var (
 			"/v2.ComplianceResultsService/GetComplianceProfileClusterResults",
 		},
 	})
+
+	log = logging.LoggerForModule()
 )
 
 // New returns a service object for registering with grpc.
-func New(complianceResultsDS complianceDS.DataStore, scanConfigDS complianceConfigDS.DataStore, integrationDS complianceIntegrationDS.DataStore, profileDS profileDatastore.DataStore, ruleDS complianceRuleDS.DataStore) Service {
+func New(complianceResultsDS complianceDS.DataStore, scanConfigDS complianceConfigDS.DataStore, integrationDS complianceIntegrationDS.DataStore, profileDS profileDatastore.DataStore, ruleDS complianceRuleDS.DataStore, scanDS complianceScanDS.DataStore) Service {
```

Comment:
Really needs a builder pattern at some point with the number of datastores :(

Reply:
I don't disagree necessarily, but that feels like a battle for another day.